### PR TITLE
OCPBUGS#24512: Added info on devices not supported for LVM Storage operations

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s): 4.15

Issue: [OCPBUGS-24512](https://issues.redhat.com/browse/OCPBUGS-24512)

Link to docs preview:
https://69929--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-unsupported-devices_logical-volume-manager-storage

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
Hi @suleymanakbas91 
Could you please review this PR? 
Thanks.

File: modules/lvms-unsupported-devices.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-unsupported-devices_{context}"]
+= Devices not supported by {lvms}
+
+When you are adding the paths to the devices in the `deviceSelector` section of the `LVMCluster` YAML, ensure that the devices are supported by {lvms}. If you add paths to the unsupported devices, then {lvms} excludes the devices to avoid complexity in managing logical volumes.
+
+If you do not specify any path to the device, then {lvms} adds only the unused devices that it supports.
```
Comment (suleymanakbas91) [inline modules/lvms-unsupported-devices.adoc:11]:
```suggestion
If you do not specify any path in the `deviceSelector`, then {lvms} adds only the unused devices that it supports. 
```

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: modules/lvms-unsupported-devices.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/lvms-unsupported-devices.adoc]: @suleymanakbas91 
This change has been incorporated. PTAL.
Thanks!

**Comment by sr1kar99** [comment]
@radeore 
Can you please review this PR?
Thanks.

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**